### ISSUE TITLE
Pulp doesn't like dev builds of psc

### ISSUE THREAD

**Issue by paf31** [purescript-contrib/pulp#257]
I get this error when trying to use the compiler on `master`:

```
* Unable to parse the version from psc. (It was: 0.11.0 [development build; commit: 9d9b4bb875eb53324e9fde2cb6a1738f2b96a276 DIRTY])
* Please check that the right psc is on your PATH.
```

**Comment by hdgarrood** [comment]
This is actually fixed in master but I haven't managed to put a new release of pulp out yet - I'll try to do that soon.

**Comment by hdgarrood** [comment]
Should be fixed as of 10.0.3, which I've just published.
